tests/Cargo.toml

```diff
@@ -43,7 +43,9 @@ name = "channel"
 path = "benches/channel/mod.rs"
 
 [features]
+default = ["failpoints", "testexport"]
```

Comment:
Why `testexport` is default? Will it introduce extra unnecessary code?

Reply:
Yes, it will. But this file is considered when building tests, and it's expected to produce and test those code.